packages/cashier-v2/src/lib/Transfer/Transfer.tsx

```diff
@@ -1,32 +1,53 @@
-import React, { useCallback, useEffect } from 'react';
-import { useActiveAccount, useCurrencyConfig, useTransferBetweenAccounts } from '@deriv/api-v2';
-import { Loader } from '@deriv-com/ui';
-import { TransferForm } from './components';
-import { TransferProvider } from './provider';
-
-const Transfer = () => {
-    return <TransferForm />;
+import React from 'react';
+import { useAccountLimits, useActiveAccount, useCurrencyConfig } from '@deriv/api-v2';
+import { Loader, Text } from '@deriv-com/ui';
+import { ErrorDialog } from '../../components';
+import { THooks } from '../../hooks/types';
+import { TransferForm, TransferReceipt } from './components';
+import { TransferProvider, useTransfer } from './provider';
+
+type TTransferModuleProps = {
+    accounts?: THooks.TransferAccounts;
 };
 
-const TransferModule = () => {
-    const { data: activeAccount } = useActiveAccount();
-    const { getConfig } = useCurrencyConfig();
-    const { data: transferAccounts, mutate } = useTransferBetweenAccounts();
-    const accounts = transferAccounts?.accounts;
+const Transfer = ({ isImported }: { isImported: boolean }) => {
+    const { setTransferReceipt, transferError, transferReceipt } = useTransfer();
+
+    if (transferReceipt) return <TransferReceipt {...transferReceipt} resetTransferReceipt={setTransferReceipt} />;
 
-    const requestForAccounts = useCallback(() => mutate({ accounts: 'all' }), [mutate]);
+    return (
+        <>
+            <TransferForm />
+            {/* isImported checks if the module is used as an import from the cashier-v2 library */}
+            {!isImported && (
```

Comment:
hmm, why this is needed? 
does not seem like its part of "transfer" form,

Reply:
Sorry @wojciech-deriv, did not catch your point.

Reply:
sorry, let me reprhase @aum-deriv  - what does the flag "isImported" do and why its needed?

Reply:
When the module is used inside of cashier-v2, for any API call error received, we show an error popup...
In case of other places, say in Trader's hub, we won't have error popups and we might have separate design for it.